packages/office-addin-test-server/src/testServer.ts

```diff
@@ -24,19 +24,21 @@ export class TestServer {
         this.m_resultsPromise = undefined;
         this.m_testServerStarted = false;
     }
-    
-    public async startTestServer(mochaTest: boolean = false): Promise<any>{
-        return new Promise<boolean>(async (resolve, reject) => {
+
+    public async startTestServer(mochaTest: boolean = false): Promise<any> {
```

Comment:
One thing I just noticed, this is "Promise<any>". Shouldn't this be "Promise<boolean>"?

Reply:
resolved locally - will push commit